config/kubernetes-sigs/org.yaml

```diff
@@ -72,6 +72,7 @@ members:
 - AndrewSirenko
 - andrewsykim
 - andreyvelich
```

Comment:
For org membership changes we do those separately with 1 commit per person with the commit message like “Add <foo> to Kubernetes-sigs org”.  Theres a golang tool you can use called “korg” in the root that makes it easy to add people with the right format 👍